charts/managed-serviceaccount/helm_render_test.go

```
@@ -0,0 +1,158 @@
+	if helmPath == "" {
+		helmPath = "helm"
+	}
+
+	baseArgs := []string{
+		"template",
+		"managed-serviceaccount",
+		".",
+		"--namespace",
+		releaseNamespace,
+	}
+	cmd := exec.Command(helmPath, append(baseArgs, args...)...)
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)